app/errors.py

```diff
@@ -0,0 +1,20 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str) -> None:
+        self.message = message
+        super().__init__(self.message)

```

Comment:
If you did it, move this to VaccineError, and you will avoid code duplication at `NotVaccinatedError` and `OutdatedVaccineError`